dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1121,6 +1178,118 @@
 		</ItemGroup>
 	</Target>
 
+	<Target Name="_XamarinComputeIlcCompileInputs">
+		<PropertyGroup>
+			<!-- Ask ILC to produce a static library -->
+			<NativeLib>static</NativeLib>
+		</PropertyGroup>
+
+		<!-- Remove libSystem.Net.Security.Native.a from tvOS when using NativeAOT, it's not shipped on tvOS -->
+		<ItemGroup>
+			<DirectPInvoke Remove="libSystem.Net.Security.Native" />
+			<NetCoreAppNativeLibrary Remove="System.Net.Security.Native" />
+			<NativeLibrary Remove="@(NativeLibrary)" Condition="'%(Filename)' == 'libSystem.Net.Security.Native'" />
+			<LinkerArg Remove="@(LinkerArg)" Condition="'%(Filename)' == 'libSystem.Net.Security.Native'" />
+		</ItemGroup>
```

Comment:
Just for the reference, this has been fixed via: https://github.com/dotnet/runtime/pull/87605

Reply:
Filed to look at it after this PR is in: https://github.com/xamarin/xamarin-macios/issues/18481